### PR TITLE
JIRA: ABC-181 Arm64 support

### PR DESCRIPTION
**Pull request by clusty**
This branch adds support for Apple silicon binaries.
ISPC was upgraded to a version that supports Arm64, 1.14.1

There is a standalone test target, * mac_m2* that can be ran by hand (there are very few arm OSX's available le.)

### CODE REVIEW

File: com.unity.formats.alembic/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 ### Chnages
 - Updated to Alembic version 1.7.16.
 - Added support for Stadia standalone builds.
+- Add support for Arm64 OSX.
```
Comment (sebastienduverne) [inline com.unity.formats.alembic/CHANGELOG.md:7]: I didn't really find an official way to properly write down the name of the thing but here is my suggestion after putting together some research results:
`- Added support for arm64 macOS.`

**Comment by clusty** [comment]
@cguer I'd say test as spot checks :)

File: com.unity.formats.alembic/CHANGELOG.md (hunk 1 above)
Comment (clusty) [reply com.unity.formats.alembic/CHANGELOG.md]: Fixed

File: com.unity.formats.alembic/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 ### Chnages
```
[the remark this answers is not in the document]
Comment (clusty) [reply com.unity.formats.alembic/CHANGELOG.md]: Fixed

File: .yamato/upm-ci.yml [hunk 3]
```diff
@@ -36,11 +36,23 @@ standalone_platforms:
     image: package-ci/mac:stable
     flavor: m1.mac
     runtime: standalone
-#  - name: centOS
-#    type: Unity::VM::GPU
-#    image: package-ci/centos:latest
-#    flavor: b1.large
-#    runtime: standalone
+
+bizarro_standalone_platforms:
```
Comment (judubu) [inline .yamato/upm-ci.yml:40]: 🤣

**Review by judubu — APPROVED** [review]
👍

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested on:
- 2020.2.1f1 (Windows, MacOS)

**Pull request merged**